app/src/main/java/com/openclassrooms/realestatemanager/repository/AgentRepository.kt

```diff
@@ -0,0 +1,17 @@
+package com.openclassrooms.realestatemanager.repository
+
+import androidx.lifecycle.MutableLiveData
+import com.openclassrooms.realestatemanager.model.Agent
+
+class AgentRepository {
+
+    private val agentList = listOf(
+        Agent(0, "Marc Leblanc"),
+        Agent(1, "Eric Dupont"),
+        Agent(2, "Elise Duval"),
+        Agent(3, "Henri Auger"),
+    )
+
+    fun getAgentListLiveData() = MutableLiveData(agentList)
```

Comment:
```suggestion
    fun getAgentListLiveData(): LiveData<List<Agent>> = MutableLiveData(agentList)
```

Pour ne pas l'exposer en tant que mutable